bdleon/workout-plus
# added button in single post view to delete post

#### Changes Made
1. Added file `block.svg` to `images` directory.
1. Modified file `login.js` to include `change route to /post` and functionality.
2. Modified file `Post.js` to include `delete button` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout bd-delete-fetch
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
4. 
    > once a user is logged in it should direct them to "./posts" in the url. Then the user should click on the link on the card. if the post is one that the user created you should see a delete button.

    > Once the user clicks on the delete button the post will delete and should take you back to the home page with new posts rendered.
   
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
